.gitignore

```diff
@@ -16,8 +16,12 @@ src/main/resources/docs/
 *.iml
 bin/
 
+# IO output files
```

Comment:
Are the files remove from cache already? The IO test for GitHub Actions depends on it